### ISSUE TITLE
Interest in chemical group theory component?

### ISSUE THREAD

**Issue by weisscharlesj** [bjodah/chempy#196]
Is there interest in seeing a chemical group theory component added to ChemPy?  It would primarily be a function that accepts the point group and reducible representation and returns the number of IR and Raman active modes. While the math is not complicated, it is tedious by hand. I'm experimenting with a solution in Python but want to know there is interest and no one else is working on it before committing to transcribing all the character table data into Python.

**Comment by bjodah** [comment]
Sure! Sounds interesting. Would you mind elaborating a bit on the nature of the functionality? It would take input in Schoenflies notation? and dipole moments? (you mention IR active modes) and polarizabilities? (for Raman active modes).

**Comment by weisscharlesj** [comment]
Yes, it would take the point group Shoenflies notation along with the reducible representation and return the number of each reducible representation (example shown below). 

    solve_reducible('C2v', [9, -1, 3, 1]) → {'A1': 2, 'A2': 2, 'B1': 2, 'B2': 2}

As I type this, I realize it would be simpler to calculate the IR and Raman active modes using separate functions that accept the above calculated number of reducible representations and return a dictionary of how many of each active mode is present. It would look something like below, so this means there are three IR active modes, two with A1 symmetry and one with B1 symmetry.

    IR_modes( {'A1': 2, 'A2': 2, 'B1': 2, 'B2': 2}) → {'A1': 2, 'A2': 0, 'B1': 1, 'B2': 0}

**Comment by bjodah** [comment]
I see, sure, I think that would be a great fit for ChemPy!
